docs/pages/web/sheetmobile.js

```diff
@@ -0,0 +1,326 @@
+import fullSize from '../../examples/sheetmobile/fullSize.js';
+import main from '../../examples/sheetmobile/main.js';
+import navigationHeader from '../../examples/sheetmobile/navigationHeader.js';
+import outsideClick from '../../examples/sheetmobile/outsideClick.js';
+import textOnlyHeader from '../../examples/sheetmobile/textOnlyHeader.js';
+import withPrimaryActionHeader from '../../examples/sheetmobile/withPrimaryActionHeader.js';
+
+export default function SheetMobilePage({
+  generatedDocGen,
+}: {|
+  generatedDocGen: {| [string]: DocGen |},
+|}): Node {
+  return (
+    <Page title={generatedDocGen?.SheetMobile.displayName}>
+      <PageHeader
+        badge="pilot"
+        name={generatedDocGen?.SheetMobile.displayName}
+        description={generatedDocGen?.SheetMobile.description}
+        slimBanner={
+          <SlimBanner
+            type="warning"
+            iconAccessibilityLabel="Warning message"
+            message="SheetMobile is in pilot phase. Expect development and design iteration and breaking API changes as well as further documentation development."
+          />
+        }
+      >
+        <SandpackExample code={main} hideEditor name="Main SheetMobile example" />
+      </PageHeader>
+
+      <GeneratedPropTable generatedDocGen={generatedDocGen?.SheetMobile} />
+
+      <MainSection name="Usage guidelines">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            title="When to use"
+            description={`
+- Use a Partial Sheet when the content of the Sheet compliments the content of the primary screen behind.
+- Use a Full Sheet when the content of the Sheet does not need to reference the primary screen behind. Usually serving as a lightweight way to complete actions or move through a flow.
+- Use in conjunction with other Gestalt components as a means to make them mobile-friendly e.g. Dropdown or Popover.`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            title="When not to use"
+            description={`
+- If possible, similar content should remain on a primary screen. Only use SheetMobile if the content is a unique experience to the primary screen and/or to reference its content.
+- To present binary or blocking decisions. Use an ModalAlert of Modal instead.
+- For a temporary message. Use Toast instead.`}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <MainSection name="Best practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            title="Do"
+            description={`
+- Always include a collapse affordance. The Sheet should close when users press the X icon button, a cancel/close button, the Esc key, when swiped away or when users click or tap the area outside the partial sheet.
```

Comment:
This is mobile-only, so clicking isn't a thing, correct?

Reply:
DONE